v3/src/components/case-table/case-table.scss

```
@@ -15,28 +15,30 @@ $table-body-font-size: 8pt;
     font-size: $table-body-font-size !important;
     color: #222222;
 
-    .rdg-header-row {
-      line-height: 14px !important;
+    .rdg-row {
+      .rdg-header-row {
```

```suggestion
    .rdg-header-row {
```
`.rdg-header-row` is not inside `.rdg-row`, so this change simply disables column header formatting.